run_in_docker.sh

```diff
@@ -35,9 +35,9 @@ if [[ ! -e $HOME/.config/gcloud/application_default_credentials.json ]]; then
 fi
 
 docker run \
-    -v $HOME/.config/gcloud:/root/.config/gcloud \
-    -v ${context}:/workspace \
+    -v "$HOME"/.config/gcloud:/root/.config/gcloud \
```

Comment:
Are the changes to this file necessary?

Reply:
Most likely not, although it's generally safer to use "" in this case. Let me know if you want me to revert @priyawadhwa

Reply:
That's alright, let's keep this! Thanks.